pycon/settings.py

```diff
@@ -337,10 +327,11 @@
     'cms_utils',
     'social_django',
 
-    # FYI using setting_locale we can't extend list of INSTALLED_APPS via +=
     # 'django_extensions',
     # 'sslserver',
 )
+# Extend INSTALLED_APPS using env settings
```

Comment:
do we need this as env setting?

Reply:
Until there's separate modules for settings, where additional settings can be defined, I can't think of another way of doing it. Once there's a separate `prod`, `staging`, `dev` and `test` settings, this can be dropped.